### PR TITLE
Poc 115

### PR DESCRIPTION
**Pull request by hiqedme**
Family enhancements

### CODE REVIEW

File: app/family-history/family-history.service.js [hunk 1]
```diff
@@ -92,33 +89,74 @@ export class FamilyTestingService {
...
-          ' tx.*, extract(year from (from_days(datediff(now(),tx.birthdate)))) as age, tx.gender as index_gender ' +
+          columns +
           from +
-          where;
+          where +
+          ' and obs_group_id is not null and t1.relationship_type not in ("SIBLING","PARTNER-SPOUSE","PARTNER-OTHER","FELLOW-WIFE","CHILD","MOTHER","FATHER") AND fm_name is not null LIMIT 2000 OFFSET ' +
+          params.startIndex;
       } else {
-        sql = sql + columns + from + where + ' and obs_group_id is not null ';
+        sql =
```
Comment (drizzentic) [inline app/family-history/family-history.service.js:147]: Might want to use shorthand appending i.e `sql+=`

File: app/family-history/family-history.service.js [hunk 2]
```diff
@@ -368,4 +412,237 @@ export class FamilyTestingService {
...
+        w = w + `  and tx.child_status_reason = 11890`;
+        break;
+      case '0':
+        w = w + `  and tx.child_status_reason = 11891`;
+        break;
+    }
+
+    sql =
+      sql +
+      //  ' tx.*, t8.status, extract(year from (from_days(datediff(now(),tx.birthdate)))) as age, tx.gender as index_gender, 1 as hideContactColumns ' +
```
Comment (drizzentic) [inline app/family-history/family-history.service.js:442]: Do we need this code here?

File: app/family-history/family-history.service.js [hunk 3]
```diff
@@ -368,4 +412,237 @@ export class FamilyTestingService {
...
+        w = w + `  and tx.child_status_reason = 11890`;
+        break;
+      case '0':
+        w = w + `  and tx.child_status_reason = 11891`;
+        break;
+    }
+
+    sql =
+      sql +
+      columns + // ' tx.*,t8.status,extract(year from (from_days(datediff(now(),tx.birthdate)))) as age, tx.gender as index_gender, 1 as hideContactColumns ' +
```
Comment (drizzentic) [inline app/family-history/family-history.service.js:476]: Consider removing any commented code.

File: package-lock.json
Comment (corneliouzbett) [inline package-lock.json:1]: I don't what's happening here but these changes are unnecessary since you haven't changed a thing in the `package.json`

**Review by corneliouzbett — CHANGES_REQUESTED** [review]

**Review by corneliouzbett — APPROVED** [review]

**Pull request merged**